### DISCUSSION TITLE
Session File

### DISCUSSION THREAD

**Discussion by brxndonnn** [new92/igfi#53] [Q&A] [answered]
How do i create a session file?

**Comment by new92** [comment, marked as answer]
Hey @brxndonroux, to create a session file simply enter the command:
`python3 cookies.py`. But before that make sure you're logged in to your account on Instagram from Firefox.

**Reply by brxndonnn** [reply]
oh so i need to use firefox?

**Reply by new92** [reply]
Yeah yeah
